tests/integration/targets/aws_eks_fargate_profile/tasks/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: 'eks_cluster integration tests'
+  collections:
+    - amazon.aws
+    - amazon.community
```

Comment:
So it would be "community.aws" but I don't think you need that line as it's already running in the context of the collection

Reply:
my mistake, i will remove this line